Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/ThrowDaggersInBensEyes.java

```diff
@@ -44,14 +44,14 @@ public void initDefaultCommand() {
         //setDefaultCommand(new MySpecialCommand());
     }
 
-	public void reverse(Object object) {
+	//public void reverse(Object object) {
 		// TODO Auto-generated method stub
 		
-	}
+	//}
 
-	public void forward(Object object) {
+	//public void forward(Object object) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)